server/routes/activities/manage-allocations/createRoutes.ts

```diff
@@ -55,5 +57,7 @@ export default function Index({ activitiesService, prisonService, metricsService
   post('/cancel', cancelHandler.POST, ConfirmCancelOptions)
   get('/confirmation', confirmationHandler.GET, true)
 
+  get('/error/:errorType(transferred)', errorHandler.GET, true)
```

Comment:
If required this can be extended to support other types of errors during allocation.